microcosm_logging/decorators.py

```diff
@@ -0,0 +1,18 @@
+"""Decorator library for common logging functionality."""
+from logging import DEBUG, getLogger
+
+
+def logger(obj):
+    """
+    logging decorator, assigning an object the `logger` property.
+    Can be used on a Python class, e.g:
+
+        @logger
+        class MyClass(object):
+            ...
+
+
+    """
+    obj.logger = getLogger(obj.__name__)
+    obj.logger.setLevel(DEBUG)
```

Comment:
not sure if u want this here by default or have some better way u thought of doing it via the whole graph / dictConfig stuff - this is just a simple self-contained operator